Install-Under-CentOS_LAMP.md

```diff
@@ -1,17 +1,26 @@
-# Installing on CentOS 7
+# Installing on CentOS/RHEL/ROCKY
 
 ## LAMP (Linux, Apache, MySQL/MariaDB, PHP) Required packages
 
 ### Web Server (Apache)
 
-1. Enable Epel repo to enable PHP 7.2 package download
+1. Enable Epel repo to enable PHP 7.2 package download ( 7.x and Below)
 
    ```console
    yum install <http://rpms.remirepo.net/enterprise/remi-release-7.rpm> -y
    yum install yum-utils -y
    yum-config-manager --enable remi-php72
    ```
 
+### For Centos/RHEL/ROCKY 8+
```

Comment:
This should not be a sub heading, consider using sub items, eg, 

```md
1. Enable Epel repo to enable PHP 7.2 package download 
   1.1 For CentOS 7.x/RHEL 7.x and below:
   ```console
   blah

   ...

   1.2 For CentOS 8.x/RHEL 8.x and above:
   ```console
   blah
```